lib/spack/llnl/util/cpu/microarchitectures.json

```diff
@@ -1174,7 +1177,8 @@
           },
           {
             "versions": "5:",
-            "flags": "-mcpu=thunderx2t99"
+            "name": "thunderx2t99",
```

Comment:
I decided to just re-use the clang arch names for Rust rather than try to define a complete separate list of flags. So therefore, the names must be exposed via the "name" key. Let me know how you feel about this approach.